examples/todomvc/components/pageview.go

```diff
@@ -15,37 +15,30 @@ import (
 )
 
 type PageView struct {
-	vecty.Composite
+	*vecty.Core
 
 	Items        []*model.Item
 	newItemTitle string
 }
 
-// Apply implements the vecty.Markup interface.
-func (p *PageView) Apply(element *vecty.Element) {
-	element.AddChild(p)
-}
-
-func (p *PageView) Reconcile(oldComp vecty.Component) {
-	if oldComp, ok := oldComp.(*PageView); ok {
-		p.Body = oldComp.Body
-		p.newItemTitle = oldComp.newItemTitle
+func (p *PageView) Reconcile(prev vecty.Component) {
+	if v, ok := prev.(*PageView); ok {
+		v.newItemTitle = v.newItemTitle
 	}
-	p.RenderFunc = p.render
-	p.ReconcileBody()
+	p.Core.Reconcile(prev)
```

Comment:
For my personal clarity / understanding, could you post a snippet of what that logic looked like before this change?

I may be wildly mistaken, but I think it would simply look something like this:

```
if renderIsNeeded {
    p.Core.Reconcile(prev)
}
```

(btw, unrelated and just a friendly note, but you are writing `loose` and `loosing`, like "these pants are too loose".. I think you mean `lose` like "we are losing the race!")